nautobot/extras/api/serializers.py

```diff
@@ -107,7 +110,29 @@
 #
```

Comment:
Please make sure that this file imports `NestedNoteSerializer` along with the other nested serializers for Extras.